bindings/python/tests/test_parse.py

```diff
@@ -404,9 +404,11 @@ class aClass {
     assert default_constructor["kind"] == "CONSTRUCTOR"
     assert default_constructor["name"] == "aClass"
     assert default_constructor["result_type"] == "void"
+    assert default_constructor["is_default_constructor"]
 
     delete_constructor = class_decl["members"][1]
 
     assert delete_constructor["kind"] == "CONSTRUCTOR"
     assert delete_constructor["name"] == "aClass"
     assert delete_constructor["result_type"] == "void"
+    # no check available for deleted ctor analogous to `is_default_constructor`
```

Comment:
Any other way we can detect? Is `delete` keyword present somewhere in the ast?

Reply:
Probably still not exposed from the C API as per [link](https://reviews.llvm.org/D15469). tl;dr: it was failing tests and should've been added as a follow-up patch, but it seemingly never did. if `delete` is not visible in the tree, LGTM

Reply:
Thanks for digging the link. Lgtm 

We'll have to deal with the issues when they arise in case or deleted copy ctor

Reply:
> Probably still not exposed from the C API as per link.

Couldn't find `clang_CXXMethod_isDeleted` in the C++ API too:  https://clang.llvm.org/doxygen/Index_8h.html

Reply:
`delete` is available in the AST dump, though: https://godbolt.org/z/5Wosoc

Reply:
This PR is good to merge. I'm just concerned of issues wrt binding in some places where I've deleted copy ctors.